.github/workflows/push-pdn-environment.yaml

```diff
@@ -0,0 +1,572 @@
+name: Deploy production environment
+
+on:
+  push:
+    tags:
+      - v*
+      
+jobs:
+
+ #   build-cpp:
```

Comment:
@fagomezra 
Why keep all the comments here?

Reply:
This part, and the part of e2e it's for automated the tests. Let me know if this part we could work, or would erase.